src/utilities/formatToProposal.ts

```diff
@@ -96,6 +98,11 @@ const formatToProposal = ({
   const againstVotesWei = new BigNumber(againstVotes || 0);
   const forVotesWei = new BigNumber(forVotes || 0);
 
+  const proposalTypeName = ['Normal', 'FastTrack', 'Critical'][proposalType] as
+    | 'Normal'
+    | 'FastTrack'
+    | 'Critical';
```

Comment:
Following my previous comment, you'd then be able to use the enum here:
```ts
const proposalTypeName = ProposalTypeName[proposalType];
```

Reply:
It can't be done this way because enums can't be indexed by an integer but I can add the type and improve on this

Reply:
cool didn't know it worked both ways I thought it this only set default values. Thanks

Reply:
No worries